ignite/cmd/scaffold_chain.go

```diff
@@ -87,6 +90,7 @@ about Cosmos SDK on https://docs.cosmos.network
 	flagSetClearCache(c)
 	c.Flags().AddFlagSet(flagSetAccountPrefixes())
 	c.Flags().AddFlagSet(flagSetCoinType())
+	c.Flags().String(flagDefaultDenom, sdk.DefaultBondDenom, "default staking denom")
```

Comment:
Perhaps we should create a flag constant in Ignite? Because if we change to the SDK, will change here